renderdoc/driver/gl/wrappers/gl_framebuffer_funcs.cpp

```diff
@@ -1473,6 +1473,35 @@ void WrappedOpenGL::glInvalidateFramebuffer(GLenum target, GLsizei numAttachment
   }
 }
 
+// note: glDiscardFramebufferEXT looks the same as glInvalidateFramebuffer. Could this be a plain
+// alias?
+void WrappedOpenGL::glDiscardFramebufferEXT(GLenum target, GLsizei numAttachments,
+                                            const GLenum *attachments)
+{
+  m_Real.glDiscardFramebufferEXT(target, numAttachments, attachments);
+
+  if(m_State == WRITING_IDLE)
+  {
+    GLResourceRecord *record = NULL;
+
+    if(target == eGL_DRAW_FRAMEBUFFER || target == eGL_FRAMEBUFFER)
+    {
+      if(GetCtxData().m_DrawFramebufferRecord)
+        record = GetCtxData().m_DrawFramebufferRecord;
+    }
+    else
+    {
+      if(GetCtxData().m_ReadFramebufferRecord)
+        record = GetCtxData().m_ReadFramebufferRecord;
+    }
+
+    if(record)
```

Comment:
I was just thinking that on desktop those functions are a minor performance hint, and may well be ignored by some drivers, but on mobile due to architecture differences it can be very important. Given mobile driver quality is lower it may end up being that omitting these functions could cause differences in replay, even though it shouldn't affect anything.

It was just a suggestion though, I wanted to mention it because I didn't want you to feel like you *must* omit the serialisation just because the other functions do.